wca-guidelines.md

```diff
@@ -93,6 +93,7 @@ To be more informative, each Guideline is classified using one of the following
 - 4d++) [ADDITION] It is permitted for the puzzle to change its orientation when it is moved from the scrambler to the solving station, as long as no one is attempting to influence the randomness of the orientation (see [Regulation A2e1](regulations:regulation:A2e1)).
 - 4f+) [RECOMMENDATION] The WCA Delegate should generate sufficient scramble sequences for the entire competition ahead of time, including spare scramble sequences for extra attempts.
 - 4f++) [REMINDER] If the WCA Delegate generates any additional scramble sequences during the competition, the scramble sequences must be saved.
+- 4g+) [ADDITION] For Multi-Blind Solving, the scrambler should verify that the puzzles are scrambled correctly and that there are no duplicate scrambles.
```

Comment:
```suggestion
- 4g+) [RECOMMENDATION] For Multi-Blind Solving, the scrambler should pay special attention to duplicate scrambles.
```

I think it's better to remove that part, so we don't give the idea that checking the scrambles is a "should". 4g already states that they must do it.